tests/queries/0_stateless/03304_string_compare.sql

```diff
@@ -0,0 +1,78 @@
+-- {echoOn }
+
+select stringCompare('abc', 'abc');
```

Comment:
I think these tests are systematic but future readers will need to check all tests one-by-one to understand the principle behind them. Please help them and add comments what is being tested to each block.